lib/matplotlib/tests/test_axes.py

```diff
@@ -4011,865 +4011,191 @@ def test_subplot_key_hash():
     assert ax.get_subplotspec().get_geometry() == (5, 1, 0, 0)
 
 
-@image_comparison(['specgram_freqs.png', 'specgram_freqs_linear.png'],
-                  remove_text=True, tol=0.07, style='default')
-def test_specgram_freqs():
-    """test axes.specgram in default (psd) mode with sinusoidal stimuli"""
+@image_comparison([
+    "specgram_freqs.png", "specgram_freqs_linear.png",
+    "specgram_noise.png", "specgram_noise_linear.png",
+], remove_text=True, tol=0.07, style="default")
```

Comment:
Style nit-pick, here and below: I think this pattern of out-denting is counterproductive and impairs readability.  I hope it doesn't take hold as typical practice.

Reply:
I personally dislike out-denting *when the delimiter ends up being the sole character(s) on its own line*, which is sadly the style used by the sadly (yes, twice) popular *black* formatter -- it would give you
```
@image_comparison(
    [
        "specgram_freqs.png",
        "specgram_freqs_linear.png",
        "specgram_noise.png",
        "specgram_noise_linear.png",
    ],
    remove_text=True,
    tol=0.07,
    style="default",
)
def test_specgram():
    """Test axes.specgram in default (psd) mode."""
```
(yuck! completely ignoring semantics that are not being captured by syntax).  I would certainly object to this.

But I do think the style in this PR is a bit more regular than e.g. (A)
```
@image_comparison(
    ["specgram_freqs.png", "specgram_freqs_linear.png",
     "specgram_noise.png", "specgram_noise_linear.png"],
    remove_text=True, tol=0.07, style="default")
def test_specgram():
    """Test axes.specgram in default (psd) mode."""
```
(now you do need to edit more things when adding new items to the list -- though the argument is not *very* strong)
or (B)
```
@image_comparison(["specgram_freqs.png", "specgram_freqs_linear.png",
                   "specgram_noise.png", "specgram_noise_linear.png"],
                  remove_text=True, tol=0.07, style="default")
def test_specgram():
    """Test axes.specgram in default (psd) mode."""
```
(losing ~10 characters worth of width on the left of the screen is just silly)

I don't mind changing everything to (A) if we can agree on that, though.

Reply:
There are many alternative patterns that would, in my view, fit comfortably with Python's basic principle of indentation.  If (A) works for you, it is OK for me.

Reply:
done